packages/clay-tooltip/src/TooltipProvider.tsx

```diff
@@ -66,6 +66,20 @@ const reducer = (state: IState, {type, ...payload}: IAction): IState => {
 	}
 };
 
+function closestParent(node: HTMLElement, s: string) {
+	const el = node;
+	let ancestor: HTMLElement | null = node;
+
+	if (!document.documentElement.contains(el)) return null;
```

Comment:
Just out of curiosity would this cover the case when the element disappeared from the screen or is it related to another use case?

Reply:
hmm not totally sure, I copied this bit of code from an existing polyfil that I cound for `Element.closest()`